plugins/chat/assets/javascripts/discourse/components/chat-thread.js

```diff
@@ -44,6 +44,8 @@ export default class ChatThreadPanel extends Component {
 
   @action
   loadMessages() {
+    this.thread.messagesManager.clearMessages();
```

Comment:
This was causing a rerender when loading a thread you already have loaded once before